tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -624,6 +632,9 @@ describe('util/extension-runners/chromium', async () => {
     });
 
     const connectClient = async () => {
+      if (!runnerInstance.wss) {
```

Comment:
Similarly there is going to be a runtime error anyway at `runnerInstance.wss.address()`. Does this help with anything?

Reply:
I opted to replace this with a $FlowIgnore suppress comment.